src/safeds/ml/classical/regression/_elastic_net_regression.py

```diff
@@ -15,7 +16,13 @@
 class ElasticNetRegression(Regressor):
     """Elastic net regression."""
```

Comment:
The docstrings needs to be adjusted as well. I'd suggest to use [this](https://github.com/Safe-DS/Stdlib/pull/232/files#r1174413655).